test/Microsoft.AspNet.Mvc.TagHelpers.Test/ImageTagHelperTest.cs

```diff
@@ -82,9 +82,6 @@ public void PreservesOrderOfSourceAttributesWhenRun()
             }
         }
 
-        [ConditionalTheory]
```

Comment:
Since you removed the `[ConditionalTheory]` attribute, these tests are currently not running on any platforms. Replace
```c#
[ConditionalTheory]
// Mono issue - https://github.com/aspnet/External/issues/21
[FrameworkSkipCondition(RuntimeFrameworks.Mono)]
```
with
```c#
[Fact]
```
to make them run on all platforms.